Client/game_sa/CEntitySA.h

```diff
@@ -106,27 +80,33 @@ class XYZStore
 };
 static_assert(sizeof(XYZStore) == 0x1FC, "Invalid size for XYZStore");
 
-class CSimpleTransformSAInterface            // 16 bytes
-{
-public:
-    CVector m_translate;
-    float   m_heading;
-};
-
-class CPlaceableSAInterface            // 20 bytes
+class CEntitySAInterface : public CPlaceableSAInterface
 {
 public:
-    CSimpleTransformSAInterface m_transform;
-    CMatrix_Padded*             matrix;            // This is actually XYZ*, change later
-};
+    virtual void         Add() = 0;
+    virtual void         Add(const CRect& rect) = 0;
+    virtual void         Remove() = 0;
+    virtual void         SetIsStatic(bool isStatic) = 0;
+    virtual void         SetModelIndex(std::uint32_t model) = 0;
+    virtual void         SetModelIndexNoCreate(std::uint32_t model) = 0;
+    virtual void         CreateRwObject() = 0;
+    virtual void         DeleteRwObject() = 0;
+    virtual CRect        GetBoundRect() = 0;
```

Comment:
`const`?

Reply:
I can't be sure with virtual methods. They are copied from gta-reversed-modern project

Reply:
Do the overrided methods modify base class's members?

Reply:
i can't guarantee that for all classes